templates/server/src/main/resources/archetype-resources/core/src/test/java/__packageInPathFormat__/general/common/base/test/DbTestHelper.java

```diff
@@ -1,44 +1,37 @@
 package ${package}.general.common.base.test;
 
 import javax.inject.Named;
-
-import org.flywaydb.core.Flyway;
+import com.devonfw.module.test.common.base.clean.TestCleanerPlugin;
+import com.devonfw.module.test.common.base.clean.TestCleanerPluginLiquibase;
+import com.devonfw.module.test.common.base.clean.TestCleanerPluginFlyway;
 
 /**
  * This class provides methods for handling the database during testing where resets (and other operations) may be
  * necessary.
  */
 @Named
-public class DbTestHelper {
+public class DbTestHelper{
 
-  private Flyway flyway;
+  
+  private TestCleanerPlugin testCleanerPlugin;
```

Comment:
Use `@Inject`. You should get familiar with DI when doing Java coding in devon4j:
https://github.com/devonfw/devon4j/blob/develop/documentation/guide-dependency-injection.asciidoc